src/utils/feature-flags.ts

```diff
@@ -21,7 +21,6 @@ const defaultFlagSet: LDFlagSet = {
   'supported-business-summary-entities': [],
   'supported-consent-amalgamation-out-entities': [],
   'supported-consent-continuation-out-entities': [],
-  'supported-continuation-in-entities': [],
```

Comment:
Wasn't used.

... Which makes sense because this isn't where continuation applications start. This is where they end (as new businesses).